app/src/main.js

```diff
@@ -7,33 +7,57 @@ const serviceConfig = require('./config/service');
 const smmtConfigLoader = require('./config/smmtConfigurationLoader');
 const smmtClientFactory = require('./smmt/client');
 const loggerFactory = require('./logger/createLogger');
+const responseCode = require('./smmt/responseCode').code;
 
 const app = express();
 app.disable('x-powered-by');
 const traceHeaderName = 'x-mot-trace-id';
 
+function processSmmtResponse(make, vin, recall, res, logger) {
+  if (recall.success) {
+    logger.info({ context: { make, vin, recall } }, 'Recall fetched successfully.');
+    res.status(200)
+      .send({
+        status_description: recall.description,
+        vin_recall_status: recall.status,
+        last_update: recall.lastUpdate,
+      });
+  } else if (recall.actionCode === responseCode.smmtInvalidMarque) {
+    const context = {
+      make, vin, errors: recall.errors,
+    };
+
+    logger.error({ context }, 'Recall fetching from SMMT failed. Invalid make.');
+    res.status(422).send({
```

Comment:
Think it would be nice to document the new http status behaviour in our Readme

Reply:
I forgot to push updated doc. done